### DISCUSSION TITLE
Custom log location stops working after container restart

### DISCUSSION THREAD

**Discussion by yurividal** [ZoeyVid/NPMplus#1235] [Q&A] [answered]
I have set this in the advanced settings of one of my proxy hosts:
`access_log /data/nginx/vaultwarden-access.log;`

Initially, it works fine, and the logs are printed in that file.

When i restart npmPlus container, this host goes back to outputting logs to `/data/nginx/access.log` instead of the one specified in the advanced settings.

**Comment by Zoey2936** [comment]
yes they are removed: https://github.com/ZoeyVid/NPMplus/blob/7c3d9947e98afb443b4678b9870a11e121be4cd6/rootfs/usr/local/bin/start.sh#L636
because of migration, if I don't remove them and someone tries toi migrate from NPM to NPMplus it would crash

**Reply by Zoey2936** [reply]
why do you need an extra log file for vaultwarden?

**Reply by yurividal** [reply]
i like to keep the logs separate , it makes it easier to sort on my log aggregator (grafana/loki) and for crowdsec log parsing.

**Reply by Zoey2936** [reply]
I can't say something about grafana/loki, but crowdsec docs are written for one log file. I don't see why it shouldn't work also with one log file

**Reply by Zoey2936** [reply, marked as answer]
how important are multiple log files for you?

**Reply by Dark4Codrutz** [reply]
sorry for necroing, but I came across this discussion ...

What about use cases where we need to disable access_log for a custom location ?
For eg when there is massive traffic that we don't care about like monitoring agents sending data in push mode.

**Reply by Zoey2936** [reply]
You can always configure logs yourself

**Reply by Dark4Codrutz** [reply]
After re-saving the proxy host config it seems that `access_log off;` is being honoured.

The start.sh script has changed since 2024, there are no more sed lines removing access_log lines.

Sorry for the inconvenience.
